docs/Testing-presubmit-Engine-PRs-with-the-Flutter-framework.md

```diff
@@ -4,16 +4,26 @@ This documentation describes how to run flutter/flutter presubmit checks on flut
 
 ## Overview
 
-1. Enable the web SDK presubmits.
+1. Create your engine pull request with the `test: all` label.
 2. Wait for all presubmit checks on your flutter/engine PR to be green.
 3. Determine the commit hash for your flutter/engine PR.
 4. Create and upload a flutter/flutter PR, (OR run tests locally).
 5. Wait for flutter/flutter presubmits/tests to run ☕.
-6. Edit `.ci.yaml` to uncomment the `runIf` block you commented out above.
 
-## 1. Edit .ci.yaml
+## 1. Create your engine pull request with the `test: all` label.
 
-Edit [`.ci.yaml`](https://github.com/flutter/engine/blob/main/.ci.yaml) to comment out the `runIf:` block in `linux_web_engine`. This will ensure Flutter Web artifacts build, otherwise most framework tests will fail during precache.
+When creating your PR, add the `test: all` label *before* submitting the PR to
```

Comment:
What does "before submitting the PR to presubmit" mean?

Reply:
That time when you're still creating the PR but before you hit the big green create button (or select `submit` in the `gh` tool) that triggers the presubmits.

There's got to be a better name for that state, but I struggled for a moment before coming up with this phrasing, which admittedly is pretty awkward.

Reply:
Reworded as "before you submit to CI" which is maybe a bit clearer?